blackbox/docs/admin/system-information.rst

```diff
@@ -1826,6 +1826,71 @@ node.
 
    The ``sys.allocations`` table is subject to :ref:`shard_table_permissions`.
 
+.. _sys-segments:
+
+Segments
+========
+
+The ``sys.segments`` table contains information about the Lucene segments
+of a shards.
+
+The segment information is useful to understand the behaviour of the underlying
+Lucene file structures for troubleshooting and performance optimization
+of shards.
+
++-------------------+-------------------------------------------+-------------+
+| Column Name       | Description                               | Return Type |
++===================+===========================================+=============+
+| ``table_schema``  | Schema name of the table of the shard.    | ``TEXT``    |
++-------------------+-----------+-------------------------------+-------------+
+| ``table_name``    | Table name of the shard.                  | ``TEXT``    |
++-------------------+-----------+-------------------------------+-------------+
+| ``node_id``       | ID of the node on which the shard resides.| ``TEXT``    |
```

Comment:
this should be `_node` same as e.g. here: https://crate.io/docs/crate/reference/en/latest/admin/system-information.html#table-schema